addons/mail/static/src/discuss/call/common/blur_performance_warning.js

```diff
@@ -0,0 +1,52 @@
+import { CallPopover } from "@mail/discuss/call/common/call_popover";
+
+import { Component, useState, useEffect } from "@odoo/owl";
+
+import { isMobileOS } from "@web/core/browser/feature_detection";
+import { useService } from "@web/core/utils/hooks";
+import { _t } from "@web/core/l10n/translation";
+
+export class BlurPerformanceWarning extends Component {
+    static template = "discuss.BlurPerformanceWarning";
+    static props = {};
+    static components = { CallPopover };
+
+    setup() {
+        this.rtc = useService("discuss.rtc");
+        this.store = useService("mail.store");
+        this.isMobileOS = isMobileOS();
+        this.state = useState({
+            showWarning: this.rtc.state.isHwAccelerationEnabled,
+        });
+        useEffect(
+            () => {
+                if (
+                    this.store.settings.useBlur &&
+                    !this.rtc.state.isHwAccelerationEnabled &&
```

Comment:
I don't think it even needs to be stored on rtcService's state, if you import the function from the blur manager in this file, it will only be evaluated at this step here if `useBlur` is true (and since it's now memoize it will just act as a constant boolean for the rest of the time too).

And it could be removed from the dependencies of the `useEffect` since it wont change